client/src/components/quiz/api/Lab7/alterationQuizQuestions.js

```diff
@@ -1,92 +1,105 @@
 const alterationQuizQuestions = [
-    {
```

Comment:
Quiz component is completely changed, probably out of scope for this story but just keep that in mind going forward, the config format is changed

Reply:
Updated in #144